guides/introduction/Embedded Schemas.md

```diff
@@ -1,89 +1,88 @@
 # Embedded Schemas
 
-Embedded schemas are a feature in Ecto that allows you to define and validate structured data. This data can reside in-memory through structs or, with a migration, be stored in a database field. Embedded schemas work with maps and arrays.
+**opening paragraph option 2**
+Embedded schemas are schemas without persistence. In fact, you can think of `embedded_schema` as `persistence_agnostic_schema`. (Using an embedded schema does not _prevent_ you from persisting the data, it just doesn't concern itself with that task.) This makes embedded schemas ideal for scenarios where you want to manage structured data without necessarily persisting it. For example, when you want an intermediate data structure between your UI layer and your data layer, or when you want to independently manipulate embedded structs as sub-schemas inside a parent schema. Some use cases for embedded schemas include:
 
-They're great for:
+**opening paragraph option 1; a 'spiritual reversion' to the original**
+Embedded schemas allow you to define and validate structured data. This data can live in memory, or can be stored in the database. Some use cases for embedded schemas include:
```

Comment:
I have created two options. Option 1 is inline with the original opening. Option 2 will probably just be deleted soon, it's a wilder opening based on my interpretation of the readings you gave me. I want to at least get your thoughts on it, even though it's probably a bad opening. Intros are notoriously hard, so will probably just save the final edits for last.

Reply:
Option 2 brings good points but that's not how I would start the discussion with. Rather, when you move the embedded schema to its own module, I would have a paragraph explaining that `embedded_schemas` in fact do not have to live inside another schema at all, and they are great for when you don't need to persist the data anywhere too.

Reply:
going with option 1 and bringing the option 2 info to the "extraction section